src/ocean/io/select/selector/TimeoutSelectedKeysHandler.d

```diff
@@ -143,7 +144,8 @@ class TimeoutSelectedKeysHandler: SelectedKeysHandler
 
                     verify(client !is null);
 
-                    if (!bsearch(cast (void*) client, timed_out_clients.ptr,
```

Comment:
Why is this change necessary btw?

Reply:
Because the second parameter is `inout(void*)` but we pass a typed parameter (`Timeout*` IIRC). Apparently DMD cannot figure out the conversion.